tests/test_spec_examples.py

```diff
@@ -39,8 +39,7 @@
 #             text = text.replace('\u21d4', '')    # byte order mark
 #             text = text.replace('\u2192', '\t')  # right arrow
 #             text = text.replace('\u00b0', '')    # empty scalar
-#             with open('tests/yaml-1.2-spec-examples/%s' % id, 'w',
-#                       encoding='utf-8') as g:
```

Comment:
Note that `encoding='utf-8'` is the default.

Reply:
You're right about UTF-8, I will revert that:
https://docs.python.org/3/library/functions.html#open